src/cdk/testing/testbed/unit-test-element.ts

```diff
@@ -78,15 +78,19 @@ export class UnitTestElement implements TestElement {
     await this._stabilize();
   }
 
-  async click(...args: number[]): Promise<void> {
-    const {left, top, width, height} = await this.getDimensions();
-    const relativeX = args.length ? args[0] : width / 2;
-    const relativeY = args.length ? args[1] : height / 2;
-
-    // Round the computed click position as decimal pixels are not
-    // supported by mouse events and could lead to unexpected results.
-    const clientX = Math.round(left + relativeX);
-    const clientY = Math.round(top + relativeY);
+  async click(...args: ['center'] | [number, number]): Promise<void> {
+    let clientX: number | undefined = undefined;
+    let clientY: number | undefined = undefined;
+    if (args.length) {
```

Comment:
What is the scenario where args.length is 0? From the method signature, shouldn't args always have at least one el?

Reply:
One of the signatures is `click()` but I have no idea why typescript didn't complain that this implementation isn't compatible with that signature. I updated it to include `[]`